### ISSUE TITLE
Security: db should only be accessible by this applet

### ISSUE THREAD

**Issue by wiiznokes** [wiiznokes/cosmic-clipboard-manager#23]
How can we do this?

**Comment by ruby232** [comment]
We can encrypt data saved in database o all database. 
I think it is better to encrypt content of clipboard only.

**Comment by wiiznokes** [comment]
I don't think this will be necessary, since applet will be packaged with flatpak in the future

**Comment by KaiserCalm** [comment]
> I don't think this will be necessary, since applets will be packaged with flatpak in the future

I'm currently using this package through the NixOS COSMIC DE flake and plan to do so later, thus a Flatpak only solution would not help in my scenario.

**Comment by wiiznokes** [comment]
> I'm currently using this package through the NixOS COSMIC DE flake and plan to do so later, thus a Flatpak only solution would not help in my scenario.

Aren't nix os packages containerized, to not conflict each others ?

**Comment by wiiznokes** [comment]
> Aren't nix os packages containerized, to not conflict each others ?

seems like not 
![Capture d’écran du 2024-10-07 19-19-07](https://github.com/user-attachments/assets/62677cd0-ec9c-40ee-bd3c-fb95a0fd2b97)

how ever, idt i will support encryption just for nix-os. Plus it will not be has perfect as flatpak (one app could still corrupt the db, and how to we store the key ? (probably with keyring system but i very much don't like it))
